test/shellcheck_failing_output.txt

```diff
@@ -0,0 +1,76 @@
+============================= test session starts ==============================
```

Comment:
This file was to show the failures to people who didn't have access to run the tests yet.  Travis fixes this too...

Now that we're green it could be deleted.